src/main/resources/com/nirima/jenkins/plugins/docker/builder/DockerBuilderPublisher/config.jelly

```diff
@@ -39,4 +39,11 @@
         <f:checkbox default="false"/>
     </f:entry>
 
+    <f:entry title="${%Disable caching}" field="noCache">
+        <f:checkbox default="false"/>
+    </f:entry>
```

Comment:
While I normally dislike "unnecessary" blank lines ... in this file (and in most of the jelly files in this plugin) the trend is to have a blank line between entries.
i.e. please add a blank line between line 44 and 45.